awx/main/scheduler/task_manager.py

```diff
@@ -283,38 +283,12 @@ def start_task(self, task, rampart_group, dependent_tasks=None, instance=None):
                 task.send_notification_templates('running')
                 logger.debug('Transitioning %s to running status.', task.log_format)
                 schedule_task_manager()
+            # at this point we already have control/execution nodes selected for the following cases
             elif rampart_group.is_container_group:
                 task.instance_group = rampart_group
```

Comment:
you might as well remove the `elif` because there's no difference between that and the `else`

Reply:
done